cicd/iqe_pod/create_iqe_pod.py

```diff
@@ -187,7 +218,7 @@ def _create_conf_secret(namespace):
         "metadata": {"name": SECRET_NAME},
         "data": {"settings.local.yaml": encoded_conf},
     }
-    oc("create", f="-", n=namespace, _in=json.dumps(secret))
+    oc("apply", f="-", n=namespace, _in=json.dumps(secret))
```

Comment:
This change is not necessary, just that using `apply` is more useful when doing a repeated deploy from command line. 

Same with L204 `_create_pod`